core/src/main/scala/org/apache/pekko/persistence/jdbc/state/SequenceNextValUpdater.scala

```diff
@@ -60,3 +60,31 @@ import slick.sql.SqlStreamingAction
 
   def getSequenceNextValueExpr() = sql"""#$nextValFetcher""".as[String]
 }
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[jdbc] class SqlServerSequenceNextValUpdater(profile: JdbcProfile,
+    val durableStateTableCfg: DurableStateTableConfiguration)
+    extends SequenceNextValUpdater {
+
+  import profile.api._
+
+  final val nextValFetcher = s"""SELECT NEXT VALUE FOR ${durableStateTableCfg.columnNames.globalOffset}""".stripMargin
```

Comment:
sql with string template..... :(

Reply:
Should it be no problem? SQL will be pre-compiled below.

Reply:
this doesn't need stripMargin call

Reply:
I think that maybe not safe with string template for sql